typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -407,6 +404,9 @@ export function updateTokenBalanceMetrics(
         token.symbol,
         Object.keys(tokenConfig) as ChainName[],
       ),
+      related_chain_names: Object.keys(tokenConfig)
```

Comment:
we should make sure this ordering is deterministic, I don't think Object.keys gives any guarantees -- e.g. by sorting like what we do on the warp IDs. I want to avoid us ending up using different labels for the same warp route if this ordering were to change

also I take it you felt it was preferable to create a new label instead of using the existing `warp_route_id`? it's not a huge deal but we do duplicate this data